viewer/multies.js

```diff
@@ -20,44 +20,45 @@
 
 /// / Modules
 /// ///////////////////////////////////////////////////////////////////////////////
+let Config, express, async, ESC, http, https, fs, path;
+
 try {
-  var Config = require('./config.js');
-  var express = require('express');
-  var async = require('async');
-  var URL = require('url');
-  var ESC = require('elasticsearch');
-  var http = require('http');
-  var https = require('https');
-  var fs = require('fs');
-  var path = require('path');
+  Config = require('./config.js');
```

Comment:
This block seems wrong? What was wrong with var.

Reply:
What's wrong with this block? `var` is not useful in most situations and harmful in many, especially with large code bases where scope is very important as there may be many variables with the same name. 

Here, it kind of makes sense. So I can add an inline eslint exclusion. Going forward, I'd like to require that `var` not be used (unless completely necessary).

Reply:
I was "taught" that every variable should have var/let/const in front of it. If you don't have var then is it stuck inside the try block?  btw multies --viewer tests are failing

Reply:
It's not stuck inside the try block since they are all initialized by `let` above. 

This is a matter of syntax preference, they both behave the same in the end:
- With `var`, the variable is hoisted to top scope. 
- With `let`, it is initialized in the top scope and defined in the try block. 

I'm happy either way since they are effectively the same solution. The `let` syntax is simply what eslint favors.

However, I do not want to disable requiring `const` and `let` globally. It is my opinion that `var` should be used sparingly, if at all. This is an uncommon case where I think it can be utilized effectively and understandably without introducing bugs. So which of these do you prefer:
1. keep it like this
2. go back to `var` but place `eslint-disable-next-line no-var` before every instance of `var`

Again, I'm happy with either solution.

Reply:
Working on fixing tests now! 🔨

Reply:
lets just remove the try and use const. People shouldn't be building themselves anymore anyway plus we've improved things

Reply:
🥳 fancy!